### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by JominWink** [zhoujuncc1/shenjingcat#6]
Hello, after using ReLU to train CNN, shall WE continue training CNN by replacing ReLU with Clamp? What is the definition of T in Q(X)?

**Comment by JominWink** [comment]
Hope to complete the project code, thank you!

**Comment by zhangluyan9** [comment]
Yes. After training with ReLU, you shall replace it with the Clamp function and Clamp_q function. (see cq.py lines 34 to 53) T is the quantization level.
